### DISCUSSION TITLE
spacy-clausie problem

### DISCUSSION THREAD

**Discussion by Abelcanc3rhack3r** [explosion/spaCy#11991] [Help: Installation]
Hi,
I have a problem running spacy-clausie:
https://spacy.io/universe/project/spacy-clausie

I installed spacy-clausie by running: `python -m pip install git+https://github.com/mmxgn/spacy-clausie.git`
Then I ran the code in the example:
```
import spacy
import claucy

nlp = spacy.load("en_core_web_sm")
claucy.add_to_pipe(nlp)

doc = nlp("AE died in Princeton in 1955.")

print(doc._.clauses)
```

I got the error:
Traceback (most recent call last):
  File "/home/yichen/PycharmProjects/openai_codex/NLP_labelling/dependency_rel/claucy.py", line 2, in <module>
    import claucy
  File "/home/yichen/PycharmProjects/openai_codex/NLP_labelling/dependency_rel/claucy.py", line 5, in <module>
    claucy.add_to_pipe(nlp)
AttributeError: partially initialized module 'claucy' has no attribute 'add_to_pipe' (most likely due to a circular import)

How do I solve this problem?

Thanks

**Comment by polm** [comment]
Note that clausie is a third-party package and so it's usually better to ask for help at their repo.

```
AttributeError: partially initialized module 'claucy' has no attribute 'add_to_pipe' (most likely due to a circular import)
```

This error usually means you have a file with the same name as your library, and so Python tries to load code from your file (making a circular import) rather than the library. Is your code in a file named `claucy.py`? If so you'll need to rename it.
